src/main/java/com/openlattice/pods/ConductorServicesPod.java

```diff
@@ -349,10 +336,10 @@ public PhoneNumberService phoneNumberService() {
         return new PhoneNumberService( hazelcastInstance );
     }
 
-    @Bean
-    public AssemblerQueryService assemblerQueryService() {
-        return new AssemblerQueryService( dataModelService() );
-    }
+//    @Bean
```

Comment:
is it safe to just remove this?

Reply:
should be yeah, just flagged for discussion